db/models.py

```diff
@@ -1 +1,89 @@
+from __future__ import annotations
+
 from django.db import models
+
+
+class Race(models.Model):
+    RACE_CHOICE = (
+        ("Elf", "Elf"),
+        ("Dwarf", "Dwarf"),
+        ("Human", "Human"),
+        ("Ork", "Ork")
+    )
+
+    name = models.CharField(
+        max_length=255,
+        choices=RACE_CHOICE,
+        unique=True
+    )
+    description = models.TextField(blank=True, null=True)
+
+    @classmethod
+    def race_import(
+            cls,
+            race_name: str,
+            race_description: str
+    ) -> Race:
+
+        if cls.objects.filter(name=race_name).exists():
+            return cls.objects.get(name=race_name)
+
+        return cls.objects.create(name=race_name,
```

Comment:
This will not affect the number of database queries but may improve readability. For this, use the `get_or_create()` method. Just try it 😸